app/helpers/application_helper.rb

```diff
@@ -1,10 +1,19 @@
 module ApplicationHelper
   def full_title(title = "")
     base_title = I18n.t "base_title"
-    if title.blank?
-      base_title
-    else
-      title + " | " + base_title
+    title.blank? ? base_title: title + " | " + base_title
```

Comment:
Using string interpolation 

```suggestion
    title.blank? ? base_title : "#{title}|#{base_title}"
```